src/shm_manager.h

```diff
@@ -101,7 +102,8 @@ class SharedMemoryManager {
   }
 
   template <typename T>
-  AllocatedSharedMemory<T> Load(bi::managed_external_buffer::handle_t handle)
+  AllocatedSharedMemory<T> Load(
+      bi::managed_external_buffer::handle_t handle, bool unsafe = false)
```

Comment:
Where and why do we need unsafe=true. It looks like I missed where it was being used.

Reply:
It is used in the request executor. The reason is that when response alloc is called we release the object to match the C interface of ResponseAlloc. When it is loaded with unsafe=true, it will be loaded without changing the reference count to make up for the released object.